packages/components/src/Progress/Progress.scss

```diff
@@ -40,7 +40,7 @@ $gradient-part-width: 100% / 6;
 		&.infinite {
 			background: transparent;
 			width: 100%;
-			animation: infinite-progression-keyframes 5s infinite;
+			animation: infinite-progression-keyframes 2s infinite;
```

Comment:
@ncomont I just set the value in the progress bar API. Please tell me if it sounds good to you